<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="edit_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Write someting in range the of ascii.</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>320</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_total_send">
       <property name="text">
        <string>Total Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_total_send"/>
     </item>
     <item>
      <widget class="QLabel" name="label_total_resend">
       <property name="text">
        <string>Total Resend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_total_resend"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>50</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Received data will be ploted here.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Should be an icon here.&quot;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSpinBox" name="num_des">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_send">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_des">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination ID</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>My ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_id">
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_received">
       <property name="text">
        <string> Received</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_received"/>
     </item>
     <item>
      <widget class="QLabel" name="label_failed">
       <property name="text">
        <string>Failed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_failed"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cln">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
